pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -338,11 +338,118 @@ A "create first" update strategy first creates a new machine, then deletes the o
+    kommander      git-tls             True    git-tls                         kommander-ca        Certificate is up to date and has not expired   59m
+    kommander      kommander-traefik   True    kommander-traefik-certificate   kommander-ca        Certificate is up to date and has not expired   59m
+    ```
 
-<!-- Add links to external documentation as needed -->
+1.  Run the following command to restart the affected pods:
+
+    ```bash
+    OS=linux; ARCH=amd64; curl -sSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/download/v1.7.2/cmctl-$OS-$ARCH.tar.gz
+    tar xzf cmctl.tar.gz
+    ./cmctl renew -A -l helm.toolkit.fluxcd.io/namespace=kommander
+    kubectl patch secret -n kommander traefik-forward-auth-ca-certificate --type json --patch '[{ "op": "replace", "path": "/data/ca.crt", "value": "'`kubectl get secret -n cert-manager kommander-ca -o go-template='{{index .data "ca.crt"}}'`'" }]'
+    
+    deploymentsKommander="kommander-cm kommander-traefik dex dex-dex-controller kommander-licensing-cm kube-oidc-proxy traefik-forward-auth-mgmt kommander-webhook dex-k8s-authenticator kubetunnel-webhook traefik-forward-auth"
+    for deployment in $deploymentsKommander; do
+      kubectl rollout restart deployment -n kommander $deployment
+    done
+    kubectl rollout restart statefulset -n kommander gitea
+    ```
+
+    Now you must fix the certificate issues with the Gitrepository by committing the updated secret to Git.
+
+1.  Clone the management repository:
+
+    ```bash
+    kubectl -n kommander get secret kommander-traefik-certificate -o go-template='{{index .data "ca.crt"|base64decode}}' > ca.crt && git clone -c http.sslCAInfo=$(pwd)/ca.crt https://$(kubectl -n kommander get secret admin-git-crede
+    ntials -o go-template='{{.data.username|base64decode}}:{{.data.password|base64decode}}')@$((kubectl -n kommander get cm konvoyconfig-kubeaddons -o go-template='{{if ne .data.clusterHostname ""}}{{.data.clusterHostname}}{{"\n"}}{{end}}' &&
+    kubectl -n kommander get ingress gitea -o jsonpath="{.status.loadBalancer.ingress[0]['ip','hostname']}") | head -1)/dkp/kommander/git/kommander/kommander
+    ```
+
+1.  Change into the directory containing the credentials manifest: `kommander/shared/kommander-git-repository`
+
+1.  Update the manifest file with what is stored in the cluster:
+
+    ```bash
+    kubectl -n kommander-flux get secret kommander-git-credentials -o yaml > git-credentials.yaml
+    ```
+
+1.  Delete the following fields from the `git-credentials.yaml` file:
+
+    - metadata.labels
+    - metadata.resourceVersion
+    - metadata.uid
+    - type
+
+1.  Replace the content of the field "data.ca" file with the output of this command:
+
+    ```bash
+    kubectl -n kommander get secret git-tls -o jsonpath='{.data.ca\.crt}'
+    ```
+
+<!-- Can we get an example output -->
```

Comment:
@palexster Since we are referring to replacing information based on the output of this command, I think it would be halpful to have an example output, if possible.